drydock/plugin.py

```diff
@@ -1,11 +1,84 @@
 from glob import glob
 import os
+import yaml
+import click
 import pkg_resources
 
+from tutor import env as tutor_env
+from tutor import config as tutor_config
 from tutor import hooks
 
 from .__about__ import __version__
 
+INSTALLATION_PATH = f'{os.getcwd()}/env'
+
+INIT_JOBS_SYNC_WAVE = -100
+
+
+def get_init_tasks():
+    """Return the list of init tasks to run."""
+    init_tasks = list(hooks.Filters.CLI_DO_INIT_TASKS.iterate())
+    context = click.get_current_context().obj
+    tutor_conf = tutor_config.load(context.root)
+
+    # Standarize deprecated COMMANDS_INIT and COMMANDS_PRE_INIT Filter
+    pre_init_tasks = []
+    for service, init_path in hooks.Filters.COMMANDS_PRE_INIT.iterate():
+        pre_init_tasks.append((service, tutor_env.read_template_file(*init_path)))
+
+    init_tasks = pre_init_tasks + init_tasks
+
+    standarized_commands = []
+    for service, init_path in list(hooks.Filters.COMMANDS_INIT.iterate()):
+        standarized_commands.append((service, tutor_env.read_template_file(*init_path)))
+
+
+    init_tasks.extend(standarized_commands)
+
+    templates_path = f'{INSTALLATION_PATH}/k8s/jobs.yml'
+    templates = list(yaml.load_all(open(templates_path, 'r', encoding='utf-8'), Loader=yaml.FullLoader))
```

Comment:
We can take inspiration from [this method](https://github.com/overhangio/tutor/blob/v15.3.7/tutor/commands/k8s.py#L181) to load the jobs. I don't think we need to define the variable INSTALLATION_PATH since Tutor should know where the `env` path is.